sdk/src/main/scala/io/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -1184,4 +1184,28 @@ class EthService(
       new Hash(Keccak256.hash(data))
   }
 
+
+  @RpcMethod("zen_dump")
+  @RpcOptionalParameters(1)
+  def dump(blockHashOrNumber: String, dumpFile: String, timeoutInSeconds: Int): Unit = {
+    if (!isEvmDumpEnabled)
+      throw new RpcException(RpcError.fromCode(RpcCode.ActionNotAllowed))
+
+    log.info(s"Dump timeout $timeoutInSeconds")
```

Comment:
I would log the real applied one (dumpTimeout) instead of the input one

Reply:
Done